examples/default/src/screens/apm/NetworkScreen.tsx

```diff
@@ -45,6 +46,32 @@ export const NetworkScreen: React.FC = () => {
     }
   }
 
+  async function sendRequestToUrlUsingAxios() {
+    let urlToSend = '';
+
+    if (endpointUrl.trim() !== '') {
+      urlToSend = endpointUrl;
+      console.log('Sending request to: ', endpointUrl);
+    } else {
+      // Use json placeholder URL as a default if endpointUrl is empty
+      console.log('sending request to default json placeholder');
```

Comment:
QQ: Do we use `console.log()` for production? This does not respect the log levels.

Reply:
These logs are found in the example app.